docs/source/index.rst

```diff
@@ -75,9 +75,19 @@ When the ``.fit()`` method is called, it automatically identifies and removes an
    :caption: Get Started
 
    Quickstart <self>
```

Comment:
is it more clear if we rename: "Quickstart" -> "Tutorials" ?

Reply:
The "Quickstart" is intended for the 3 pointers to quickly cover the usage of `get_noise_indices` and `LearningWithNoisyLabels`, but "Tutorials" are more in-depth specific use of an ML task. What do you think?

Reply:
I'd prefer `Tutorials` with the Quickstart tutorials simply listed as the first notebooks in this section.  It seems weird to me to have both a Quickstart section and a Tutorials section when both are just tutorial notebooks.

Reply:
I got you. Just as a final confirmation, which table of content do you prefer:

Option 1:
> **Get Started**
> 
> Tutorials
> Image Classification with Cleanlab, PyTorch & Skorch

or 

Option 2:
> **Tutorials**
> 
> Quickstart
> Image Classification with Cleanlab, PyTorch & Skorch

The current one looks like this: 
<img width="140" alt="image" src="https://user-images.githubusercontent.com/90272361/157904626-c1a341c0-7cfe-44b9-ab1a-7d79d44855a1.png">

Reply:
I prefer only `Tutorials` section.
We could just add `(Quickstart)` in the title of the notebooks that are more introductory tutorials and `(In Depth)` for tutorials that are more advanced ones. Eg:
 `Image Classification with Cleanlab, PyTorch & Skorch (Quickstart)`